xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -661,18 +665,15 @@ span#avatarUpload {
   margin: 0;
 }
 
-.attachment-picker-start {
-  background: url("$xwiki.getSkinFile('icons/silk/picture_edit.png')") no-repeat center center $theme.pageContentBackgroundColor !important;
-  border: 0 none !important;
-  border-bottom-left-radius: 8px;
-  height: 18px;
-  position: absolute;
-  right: 0;
-  text-align: left;
-  text-indent: -9999px;
-  top: 1px;
-  width: 18px !important;
-  z-index: 1;
+.attachment-picker {
+  .buttonwrapper {
+    a.button {
+        .btn;
+        .btn-default;
+        width: ~"calc(100% - 0.6em)";
+        margin: 0 0.3em 0.3em 0.3em;
```

Comment:
I was afraid that this calc would break responsiveness but apparently it's okay.

Instead of using calc and hard coded values, you can also use `display: flex` on the parent (.wikilink here) and `flex-grow` on the a.button . The button will grow to fill the whole line.

See https://css-tricks.com/snippets/css/a-guide-to-flexbox/ , flex is a really powerful tool :)

Reply:
Updated with [2b2162f](https://github.com/xwiki/xwiki-platform/pull/2862/commits/2b2162f016142b126b399d87a197b42a5de3f468)